components/emoji_picker/emoji_picker.jsx

```diff
@@ -176,6 +176,14 @@ export default class EmojiPicker extends React.PureComponent {
         this.divHeight = this.emojiPickerContainer.offsetHeight;
     }
 
+    componentDidUpdate() {
+        requestAnimationFrame(() => {
```

Comment:
This shouldn't happen on every update since it updates constantly as you scroll or mouse over an emoji. What's this code trying to accomplish here?

Also, the requestAnimationFrame shouldn't be necessary since the component should already be visible unlike when it's first mounting

Reply:
@hmhealey the code is trying to see when the search tab is selected and to focus its search box.

Reply:
As it is, it's constantly trying to refocus the search box as the user scrolls and moves their mouse around because those trigger componentDidUpdate to run because those actions change the values of `this.state.cursor` and `this.state.divTopOffset`. We need to find some state change like that to trigger the search box focus to prevent it from happening constantly